Snowflake.Data.Tests/IntegrationTests/SFMaxLobSizeSwitchIT.cs

```diff
@@ -0,0 +1,51 @@
+using System.Data;
+using NUnit.Framework;
+using Snowflake.Data.Client;
+
+namespace Snowflake.Data.Tests.IntegrationTests
+{
+    [TestFixture]
+    public class SFMaxLobSizeSwitchIT : SFBaseTest
+    {
+        [Test]
+        [Ignore("TODO: Enable when Max LOB size is available on the automated tests environment")]
+        public void TestIncreaseMaxLobSizeParameterSwitchSelect()
+        {
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+                IDbCommand cmd = conn.CreateCommand();
+
+                //setup
+                cmd.CommandText = "alter session set FEATURE_INCREASED_MAX_LOB_SIZE_IN_MEMORY='ENABLED'";
+                cmd.ExecuteNonQuery();
+                cmd.CommandText = "alter session set ENABLE_LARGE_VARCHAR_AND_BINARY_IN_RESULT=false";
+                cmd.ExecuteNonQuery();
+
+                //initial test
+                try
+                {
+                    cmd.CommandText = "select randstr(20000000, random()) as large_str";
+                    cmd.ExecuteReader();
+                }
+                catch (SnowflakeDbException e)
+                {
+                    Assert.That(e.Message, Does.Contain("exceeds supported length"));
+                }
+
+                //parameter switch
+                cmd.CommandText = "alter session set ENABLE_LARGE_VARCHAR_AND_BINARY_IN_RESULT=true";
```

Comment:
you could consider if it would be ok to create 2 tests instead. Then you could give better (more meaningful) names for them.

Reply:
IMHO there's no need, since after your suggestions the test is much shorter